### PR TITLE
Fix flexberry-edit-form blueprints

### PR DESCRIPTION
**Pull request by KollegOFF**
[TFS#104867](http://dis-tfs:8080/Common/Flex/_workitems?_a=edit&id=104867).

### CODE REVIEW

File: blueprints/flexberry-edit-form/files/__root__/routes/__name__/new.js [hunk 1]
```diff
@@ -2,5 +2,6 @@ import EditFormNewRoute from 'ember-flexberry/routes/edit-form-new';
 
 export default EditFormNewRoute.extend({
   modelProjection: '<%=modelProjection%>',
-  modelName: '<%=modelName%>'
+  modelName: '<%=modelName%>',
+  templateName: '<%=entityName%>',
```
Comment (ehaberev) [inline blueprints/flexberry-edit-form/files/__root__/routes/__name__/new.js]: I have doubts about using entityName here. May be we shold use formName? But I can mistake. So I just want to make sure we use right property.

Comment (KollegOFF) [inline blueprints/flexberry-edit-form/files/__root__/routes/__name__/new.js]: I consulted with a specialist.
I thought about using 'formName', this name form of metadata from designer.

Comment (ehaberev) [inline blueprints/flexberry-edit-form/files/__root__/routes/__name__/new.js]: So name of route and template that use for edit form are equal to 'entityName', aren't they?

Comment (KollegOFF) [inline blueprints/flexberry-edit-form/files/__root__/routes/__name__/new.js]: It [`'formName'`](https://github.com/Flexberry/ember-flexberry/blob/develop/vendor/flexberry/dummy-metadata/edit-forms/ember-flexberry-dummy-application-user-edit.json#L4), `'entityName'` set when command is invoked, example:
`ember generate flexberry-edit-form ember-flexberry-dummy-application-user-edit --metadata-dir=vendor/flexberry/dummy-metadata`

**Pull request merged**